unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,90 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test: device D400*
+
+import time
+import pyrealsense2 as rs
+from rspy import test
+
+# This test checks Timestamp domain toggle option for color and depth sensor
+
+
+def frame_callback(frame):
+    global expected_time_stamp_domain
+    test.check_equal(int(frame.get_frame_timestamp_domain()), expected_time_stamp_domain)
+
+
+################################################################################################
+test.start("Start test")
+
+ENABLE = 1
+DISABLE = 0
+
+devices = test.find_first_device_or_exit()
```

Comment:
`devices` --> `device`, we get the first device connected

Reply:
Sorry for the misunderstanding, what I meant is rename `devices` to `device`.
This function returns the first device in the list of connected devices.
No need for the new function because thats exactly what `find_first_device_or_exit()` does.
Please rollback and rename to device :)

Reply:
Oh, Ok. The function removed.